common/beerocks/hostapd/include/configuration.h

```diff
@@ -51,13 +51,29 @@ class Configuration {
     /**
      * @brief set key/value for the given vap
      * @details set the key/vale for the given vap, either replace or create.
+     * @param 
+     * - the vap to set values for (string)
+     * - the key to set its value (string)
+     * - the value (string)
      * @return 
      * true - the vap exists, values were set.
      * false - the given vap was not found
      */
     bool set_create_vap_value(const std::string &vap, const std::string &key,
                               const std::string &value);
 
+    /**
+     * @brief set key/value for the given vap
+     * @details set the key/vale for the given vap, either replace or create.
+     * @param 
+     * - the vap to set values for (string)
+     * - the key to set its value (string)
+     * - the value (int)
+     * @return 
+     * true - the vap exists, values were set.
+     * false - the given vap was not found
+     */
```

Comment:
this doesn't align with how we document the APIs elsewhere in the code

Reply:
I kindly refuse to treat comments on doxygen until [PPM-44](https://jira.prplfoundation.org/browse/PPM-44) is done.